mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -141,28 +158,17 @@ const PhaseOneBasicGamePlay = ({
                 />
                 {!submitted && (
                     <RoundButton
-                        style={
-                            (selectedAnswerIndex || selectedAnswerIndex === 0)
-                                ? styles.answerChosen
-                                : styles.submitAnswer
-                        }
+                        style={styles.submitAnswer}
                         titleStyle={styles.submitAnswerText}
                         title="Submit Answer"
                         onPress={handleSubmitAnswer}
```

Comment:
I've merged a new branch into dev that also relates to this. If you could git pull dev, you'll see `disabled` added to some buttons. The joys of working on overlapping screens!